docs/writing-tests/integrations/vitest-addon.mdx

```diff
@@ -185,56 +183,15 @@ This screenshot shows how you can run your Vitest tests in VSCode using the [Vit
 
 ### In CI
 
-For the most part, running your Storybook tests in CI is done [via the CLI](#cli). However, to have the test output link to your published Storybook on test failures, you need to provide the [`storybookUrl` option](#storybookurl) in the plugin configuration.
-
-Here's an example using GitHub Actions. The steps are similar for other CI providers, though details in the syntax or configuration may vary.
-
-When actions for services like Vercel, Netlify and others run a deployment job, they follow a pattern of emitting a `deployment_status` event containing the newly generated URL under `deployment_status.target_url`. This is the URL to the published Storybook instance. We then pass that URL to the plugin configuration using an environment variable, `SB_URL`. Finally, we update the plugin configuration to use that environment variable in the `storybookUrl` option.
-
-```yaml title=".github/workflows/test-storybook.yml"
-name: Storybook Tests
-on: deployment_status
-jobs:
-  test:
-    timeout-minutes: 60
-    runs-on: ubuntu-latest
-    if: github.event.deployment_status.state == 'success'
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
-        with:
-          node-version: '18.x'
-      - name: Install dependencies
-        run: yarn
-      - name: Run Storybook tests
-        run: yarn test-storybook
-        env:
-          SB_URL: '${{ github.event.deployment_status.target_url }}'
-```
+For the most part, running your Storybook tests in CI is done [via the CLI](#cli).
 
-```js title="vitest.workspace.ts"
-export default defineWorkspace([
-  // ...
-  {
-    // ...
-    {
-      plugins: [
-        storybookTest({
-          // ...
-          storybookScript: 'yarn storybook --ci',
-          storybookUrl: process.env.SB_URL
-        }),
-      ],
-    },
-  },
-])
-```
+However, to have the test output link to your published Storybook on test failures, you need to provide the [`storybookUrl` option](#storybookurl) in the plugin configuration. Please reference the [detailed example in the Testing in CI guide](./in-ci.mdx#21-debugging-test-failures-in-ci).
 
 ## How it works
```

Comment:
When you're able, can you take another pass at this page specifically from this section (i.e., How it works) and onto  the one below (i.e., Configuring tests), and restructure it as we're:
- Repeating some information including the testing types and how to do it in the Storybook UI.
- The user needs to read through the middle of the page to learn about the test types he/she can perform and how to toggle them.